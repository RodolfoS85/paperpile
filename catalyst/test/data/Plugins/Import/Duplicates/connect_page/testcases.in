---
file: data/Plugins/Import/Duplicates/connect_page/duplicatetest1.ppl

