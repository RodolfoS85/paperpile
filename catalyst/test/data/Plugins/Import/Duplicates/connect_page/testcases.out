---
guid: 77570396589E11E0AFBA9F3DA18D3C30
---
guid: 8AB72254589E11E0BE7B9F3DA18D3C30
